model/team_member.go

```diff
@@ -16,6 +16,12 @@ type TeamMember struct {
 	DeleteAt int64  `json:"delete_at"`
 }
 
+type TeamMemberUnread struct {
```

Comment:
I think we should exclude the `Member` part here and just go with `TeamUnread` since really it doesn't have to have anything to do with the `TeamMember`, right?